gqcp/include/QCModel/HF/UHF.hpp

```diff
@@ -292,6 +293,23 @@ class UHF {
     }
 
 
+    /**
+     *  @param K_a            the number of alpha spatial orbitals
+     *  @param K_b            the number of beta spatial orbitals
+     *  @param N_a            the number of alpha electrons
+     *  @param N_b            the number of beta electrons
+     * 
+     *  @return the implicit (i.e. with ascending and contiguous orbital indices) occupied-virtual orbital space for both the alpha and beta components that corresponds to these UHF model parameters
```

Comment:
documentation

Reply:
I checked every documentation block in the `SpinResolvedOrbitalSpace`file and apparently forgot to check the only one I added to the `QCModel` :sweat_smile: